code/game/objects/items/airbubble.dm

```diff
@@ -374,6 +371,10 @@
 		qdel(W)
 		update_icon()
 	else if(istype(W, /obj/item/weapon/wirecutters))
+		if(!zipped)
+			user.visible_message("<span class='warning'>[src]'s has no cables to cut.</span>")
```

Comment:
src's what?

Reply:
This should also be a to_chat, not vis_message.

Reply:
Same thing though. Plus this is to a user who is trying to cut it.